pyblish/cli.py

```diff
@@ -355,7 +355,7 @@ def gui(ctx, package):
 
     with _cli_plugin(data=context.data) as plugin_path:
         environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
-            ctx.obj["plugin_paths"] + [plugin_path]
+            list(ctx.obj["plugin_paths"]) + [plugin_path]
```

Comment:
This is the only important code in this PR

Reply:
- should actually be `ctx.obj["plugin_paths"] + (plugin_path, )` (or `tuple([plugin_path])` )since `ctx.obj["plugin_paths"]` is tuple and should be tuple
- I'll create PR in base